common/src/main/java/com/wynntils/screens/OverlayManagementScreen.java

```diff
@@ -313,6 +313,12 @@ public boolean keyPressed(int keyCode, int scanCode, int modifiers) {
             selectedOverlay
                     .getConfigOptionFromString("horizontalAlignmentOverride")
                     .ifPresent(configHolder -> selectedOverlay.updateConfigOption(configHolder));
+        } else if (keyCode == GLFW.GLFW_KEY_ENTER) {
+            ConfigManager.saveConfig();
```

Comment:
I think you need an ` McUtils.mc().setScreen(new OverlaySelectionScreen());` here as well. At least nothing visibly happens when you press enter.

Reply:
Right, oops.